teleport/datadog_checks/teleport/check.py

```diff
@@ -8,6 +8,8 @@
 
 METRIC_MAP = {**COMMON_METRICS_MAP, **PROXY_METRICS_MAP, **AUTH_METRICS_MAP}
 
+DEFAULT_DIAG_PORT = 3000
```

Comment:
Do you think this could be a class-level constant?

Reply:
Mmm, not sure. I'll need to use this value in tests at some point.
I'll think about it in my next PR.

Reply:
OK. But remember that it is not good practice to use production code in your tests beyond your SUT and its public interface.